contracts/adapters/MakerAdapter.sol

```diff
@@ -69,6 +75,10 @@ contract MakerAdapter {
         ManagerLike manager = ManagerLike(serviceRegistry.getRegisteredService(CDP_MANAGER_KEY));
         (uint256 cdpId, ) = decode(triggerData);
         address cdpOwner = manager.owns(cdpId);
+        require(
+            canCall(address(this), manager, cdpId, cdpOwner),
+            "maker-adapter/not-allowed-to-call"
+        ); //missing check to fail permit if msg.sender has no permissions
```

Comment:
we might want to switch ti custom errors in the entire repository - if time allows.